### PR TITLE
Publish tools to Artifactory: Blob Inspector, Network Bootstrapper

### PR DESCRIPTION
**Pull request by josecoll**
Adding publication to Artifactory of following tools:
- network bootstrapper
- blob inspector

Clean-up redundant references.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -21,7 +21,6 @@ include 'experimental:behave'
 include 'experimental:sandbox'
 include 'experimental:quasar-hook'
 include 'experimental:kryo-hook'
-include 'experimental:blobinspector'
```
Comment (shamsasari) [inline settings.gradle]: Doh!

File: tools/blobinspector/build.gradle [hunk 2]
```diff
@@ -18,11 +20,15 @@ jar {
...
     }
-    baseName 'blobinspector'
+    archiveName = "blob-inspector-${version}.jar"
     manifest {
         attributes(
                 'Automatic-Module-Name': 'net.corda.blobinspector',
                 'Main-Class': 'net.corda.blobinspector.MainKt'
         )
     }
 }
```
Comment (shamsasari) [inline tools/blobinspector/build.gradle]: In the ENT PR you had an extra `jar`: https://github.com/corda/enterprise/pull/943/files#diff-95f866e30c06a786cefa8898c67a95e7R30

File: tools/blobinspector/build.gradle [hunk 3]
```diff
@@ -18,11 +20,15 @@ jar {
...
+
+publish {
+    name 'tools-blob-inspector'
```
Comment (shamsasari) [inline tools/blobinspector/build.gradle]: Similarly there's a `disableDefaultJar = true` in ENT

**Comment by shamsasari** [comment]
Backport of https://github.com/corda/enterprise/pull/943

File: tools/blobinspector/build.gradle (hunk 2 above)
Comment (tlil) [reply tools/blobinspector/build.gradle]: not anymore; https://github.com/corda/enterprise/blob/pre-release-V3/tools/blobinspector/build.gradle

File: tools/blobinspector/build.gradle (hunk 3 above)
Comment (tlil) [reply tools/blobinspector/build.gradle]: This seem to be different though :+1:

Comment (josecoll) [reply tools/blobinspector/build.gradle]: Discussed with @Clintonio. 
That directive should not be included when publishing the default jar (so its more clean-up than anything else).

File: tools/blobinspector/build.gradle (hunk 2 above)
Comment (josecoll) [reply tools/blobinspector/build.gradle]: The ENT PR also publishes the database migration tool (which doesn't exist in OS).

Comment (tlil) [reply tools/blobinspector/build.gradle]: I think he was referring to this jar-block :)
```
jar {
    classifier "ignore"
}
```

Comment (josecoll) [reply tools/blobinspector/build.gradle]: OK - yes, so confirmed with @Clintonio that this additional is not required in this scenario.

**Pull request merged**